README.md

```diff
@@ -26,59 +26,78 @@ disable this feature, run with this environment variable:
 `ESLINT_NO_DEV_ERRORS=true`. If you always prefer not to have this behavior,
 consider setting that in your shell configuration (e.g. `.bashrc`).
 
-### Ubuntu Quickstart
+### Developing on a VM from MacOS
+We strongly recommend development in this repo on a VM running Debian. Our production machines are configured with Debian so this will allow for you to develop in the environment most similar to production. Additionally VM features such as snapshots make development much more straightforward. See the [Virtual Machine Setup Guide](./VirtualMachineSetup.md) for more details on how to best configure this VM. Then come back and follow the steps in the Debian quickstart below to get developing. 
 
-This expects Ubuntu 20.04, though it may work on other versions. This installs
-the right version of NodeJS manually. You can use a tool like
-[nvm](https://github.com/nvm-sh/nvm) or [volta](https://volta.sh) to do this in
-a nicer way.
+### Debian Quickstart
 
-```sh
-# install some prerequisites:
-sudo apt update
-sudo apt install -y git curl python
+This expects Debian 11.02, though it may work on other versions. 
 
-# change this to wherever you want:
-NODE_ROOT="${HOME}/usr/local/nodejs"
+Debian, by default, does not give your primary user account sudo access. Most of our scripts are designed assuming you will have sudo access so is most straightforward to simply add yourself to the sudoers file. You can do this with the following commands.
 
-# support x86 or ARM (e.g. Apple Silicon):
-ARCH=$(uname -m | sed 's/x86_/x/' | sed 's/aarch/arm/')
+```sh
+su - # this will prompt for the root password
+usermod -aG sudo <USERNAME> # use your user account username
```

Comment:
See my other comment, I'd replace `<USERNAME>` with `$USER` so bash will substitute it automatically.

Reply:
also it might be worth including an `exit` command to get out of the root shell.

Reply:
also I'm surprised this works, I didn't think the `sudo` group was included by default in Debian. Another option is to run `visudo` from a root shell and explicitly add a line giving the user sudo permissions.

Reply:
Since this is within the su - console if you do $USER it will be root not your other user.

Reply:
You could do this:

```sh
usermod -aG sudo $(who am i | awk '{ print $1 }')
```